python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/mpls-rsvp.j2

```diff
@@ -0,0 +1,100 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - mpls interfaces #}
+{% if mpls.rsvp is arista.avd.defined %}
+
+### MPLS RSVP
+
+#### MPLS RSVP Summary
+
+| Setting | Value |
+| ------- | ----- |
+{%     if mpls.rsvp.refresh is arista.avd.defined %}
+{%         if mpls.rsvp.refresh.interval is arista.avd.defined %}
+| Refresh interval | {{ mpls.rsvp.refresh.interval }} |
+{%         endif %}
+{%         if mpls.rsvp.refresh.method is arista.avd.defined %}
+| Refresh method  | {{ mpls.rsvp.refresh.method }} |
+{%         endif %}
+{%     endif %}
+{%     if mpls.rsvp.hello.interval is arista.avd.defined and mpls.rsvp.hello.multiplier is arista.avd.defined %}
```

Comment:
since interval and multiplier is required key so just check of mpls.rsvp.hello

Reply:
Removed required_key from both